### DISCUSSION TITLE
can't go left or right in filter text box in song select menu

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#12996] [Ideas]
songs are rewinding not text cursor, even when textbox is clicked

[database.log](https://github.com/ppy/osu/files/2329018/database.log)
[network.log](https://github.com/ppy/osu/files/2329019/network.log)
[performance.log](https://github.com/ppy/osu/files/2329020/performance.log)
[performance-gamethread.audio.log](https://github.com/ppy/osu/files/2329022/performance-gamethread.audio.log)
[performance-gamethread.draw.log](https://github.com/ppy/osu/files/2329023/performance-gamethread.draw.log)
[performance-gamethread.input.log](https://github.com/ppy/osu/files/2329024/performance-gamethread.input.log)
[performance-gamethread.update.log](https://github.com/ppy/osu/files/2329025/performance-gamethread.update.log)
[runtime.log](https://github.com/ppy/osu/files/2329026/runtime.log)

**Comment by peppy** [comment]
This matches how stable works. Do you have a proposal?

**Comment by ticotaco72** [comment]
yeah, i think that when textbox is cliked arrows should be handled by textbox until enter goes in or other thing is clicked.

**Comment by peppy** [comment]
Song select's textbox is global. It does not need to be clicked to be active. Clicking it wouldn't make much sense.

**Comment by ticotaco72** [comment]
yeah, but it still could be clickable for text rewinding, but not necessary
